### DISCUSSION TITLE
Update management z automation account

### DISCUSSION THREAD

**Discussion by rabusiak** [AzureCommunityPL/group#102] [Pomoc]
Jak w temacie, dodałem sobie testowo około 70 maszyn do update management. Można powiedzieć, że jest pełen przekrój, są vmki, fizyczne, windowsy, linuxy, azure, onprem itd. Zasadniczo działa, po jakimś czasie i kilku deploymentach wszystkie są "compliant". 

Pytania zasadnicze: 

Czy da się zmienić domyślną definicję bycia zgodnym (zainstalowane updejty critical i security)? 

Jak wymusić instalowanie others update? Update managment je rozpoznaje. Przeleciało kilka deploymentów, wszystkie success, w ustawianiach wypychanie wszystkich rodzajów updatejtów a i tak instalowane są tylko security i critical.

**Comment by smajchrz** [comment]
Delikatny wykop ale sprobuje 😄 

1.  Status complain non-complaint generowany jest na podstawie zapytania do LA ktorego nie mozna zmienic. W skrócie, nie.

2. Czy scheduler maja zaznaczone wszystkie rodaje updatow dla Linuxa? Jak wyglada raport po takim deploymencie?

**Reply by rabusiak** [reply]
Delikatny? xD
Taka jest różnica między grupą na fb a github discussions - 40 dni na jakikolwiek odzew i wiatr hula po pustych stronach ;d

Scheduler ma (i na win i na linuxach) wszytko co się tylko da do instalowania, w logach widać, że proces updatu widzi i instaluje je rzekomo,  ale nie są zainstalowane. Na linuxie instalują się tylko security i critical, reszta leży odłogiem. Na windowsach w zasadzie tylko omija windows defender definitions i net frameworkowe.
